src/plugins/vis_type_vislib/public/index.scss

```diff
@@ -1 +1,5 @@
-@import './vislib/index'
+@import './vislib/index';
+
+.vislib .visLegend__toggle {
```

Comment:
This code should live inside `src/plugins/vis_type_vislib/public/vislib/components/legend/_legend.scss`.

Reply:
00b3fc8